src/Features/CSharp/Portable/MoveToNamespace/CSharpMoveToNamespaceService.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.CSharp.MoveToNamespace
 {
     [ExportLanguageService(typeof(IMoveToNamespaceService), LanguageNames.CSharp), Shared]
     internal class CSharpMoveToNamespaceService :
-        AbstractMoveToNamespaceService<NamespaceDeclarationSyntax, TypeDeclarationSyntax>
+        AbstractMoveToNamespaceService<NamespaceDeclarationSyntax, TypeDeclarationSyntax, CompilationUnitSyntax>
```

Comment:
seriously minor nit: consider ordering from "widest to narrowest scope".  so CompilationUnit, NamespaceDecl, TyepDecl.

Reply:
We need a refactoring for that ;)